File: class2_bronze.py
# https://www.acmicpc.net/problem/4153 [직각삼각형]
import sys

while True:
    triangles = list(map(int, sys.stdin.readline().split())) #리스트에 넣은 후
    triangles.sort() #소팅한다. sort()는 본체의 list를 정렬해서 반환하고, sorted는 정렬한 새로운 list를 반환한다.
    if triangles[0] == 0:
        break
    elif ((triangles[0]**2) + (triangles[1]**2)) == (triangles[2]**2): #소팅하면 제일 큰게 마지막 idx를 가진다.
        print("right")
    else:
        print("wrong")

File: class2_bronze.js
// https://www.acmicpc.net/problem/2798 [블랙잭] - 브루트 포스(brute force)

const input = require("fs").readFileSync("/dev/stdin").toString().trim().split(/\s/).map(item => parseInt(item));

let N = parseInt(input[0]);
let M = parseInt(input[1]);
let sum = 0;
let cards = input.slice(2); //2번째 줄부터 입력받은 숫자들을 배열 형태로 저장.

for(let i = 0; i < N-2; i++){
    for(let j = i+1; j < N-1; j++){
        for(let k = j+1; k < N; k++){
            if(cards[i]+cards[j]+cards[k] <= M && cards[i]+cards[j]+cards[k] > sum){
                    sum = cards[i]+cards[j]+cards[k];
            }
        }
    }
}

console.log(sum);
